.ci/pytorch/test.sh

```diff
@@ -615,6 +615,7 @@ test_single_dynamo_benchmark() {
 }
 
 test_inductor_micro_benchmark() {
+  install_torchao
```

Comment:
We need to `TORCH_CUDA_ARCH_LIST` here in the same way as https://github.com/pytorch/pytorch/blob/main/.ci/pytorch/test.sh#L1462 to successfully build ao.

```suggestion
  if [[ "$BUILD_ENVIRONMENT" == *cuda* ]]; then
    export TORCH_CUDA_ARCH_LIST="8.0;8.6"
  fi  
  install_torchao
```